python/ray/tune/experiment.py

```diff
@@ -123,6 +123,14 @@ def __init__(self,
                  max_failures=0,
                  restore=None):
 
+        if loggers is not None:
+            # Most users won't run into this as `tune.run()` does not pass
+            # the argument anymore. However, we will want to inform users
+            # if they instantiate their `Experiment` objects themselves.
+            raise ValueError(
+                "Passing `loggers` to an `Experiment` is deprecated. Use "
+                "an `ExperimentLogger` callback instead.")
```

Comment:
Can you say what to do with the ExperimentLogger callback (i.e, pass it as a callback to the corresponding class)?